structure_tests/tests/structure_test_tests.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Copyright 2016 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+#Structure test tests. The tests to test the structure tests.
+
+#End to end tests to make sure the structure tests do what we expect them
+#to do on a known quantity, the latest debian docker image.
+
+export TEST_TAG
+TEST_TAG="test_tag-$(date +%Y-%M-%d-%H%M%S)"
+export FILE="debian_test.json"
+
+failures=0
+# build newest structure test image
+pushd ..
+./build.sh gcr.io/gcp-runtimes/structure-test-test:"$TEST_TAG"
+popd
+
+envsubst < cloudbuild.yaml.in > cloudbuild.yaml
+gcloud beta container builds submit . --config=cloudbuild.yaml
+if [ "$?" -gt "0" ]
+then
+  echo "Success case test failed"
+  failures=$((failures + 1))
+fi
+
+FILE="debian_failure_test.json"
+envsubst < cloudbuild.yaml.in > cloudbuild.yaml
+gcloud beta container builds submit . --config=cloudbuild.yaml
+if [ "$?" -ne "1" ]
+then
+  echo "Failure case test failed"
+  failures=$((failures + 1))
+fi
+
+echo "Failures: $failures"
```

Comment:
That still happens, you see the failures, I just want all the tests to run no matter how many failures there are.